percona/controller/pgcluster/controller.go

```diff
@@ -78,12 +85,13 @@ func (r *PGClusterReconciler) watchSecrets() handler.Funcs {
 			labels := e.ObjectNew.GetLabels()
 			crName := labels[naming.LabelCluster]
 
-			_, ok := labels[v2beta1.LabelPMMSecret]
```

Comment:
just to make sure, patching pmm api key still works right?

Reply:
@pooknull monitoring test should check it but please recheck